scripts/code-server.sh

```diff
@@ -5,22 +5,29 @@ set -e
 
 # Define these 2 variables if you want to customize the default installation,
 # then copy-paste the remainder of the script:
-# - the version of code-server to install
+# - the version of code-server to install (leave undefined to install the last one)
 # code_server_version=3.7.2
 # - the code-server extensions to install (space-separated names)
 # code_server_extensions="ms-azuretools.vscode-docker coenraads.bracket-pair-colorizer-2"
 # Note: install code-server after installing docker if you plan to use a docker extension
-# - the JSON content of code-server settings (here, to make the terminal a JSON shell)
+# - the JSON content of code-server settings (here, to make the terminal a JSON shell + dark theme)
 # (see https://code.visualstudio.com/docs/editor/integrated-terminal#_shell-arguments)
-# code_server_settings='{ "terminal.integrated.shellArgs.linux": ["-l"]}'
+# code_server_settings='{ "terminal.integrated.shellArgs.linux": ["-l"], "workbench.colorTheme": "Default Dark+"}'
 
 # https://github.com/cdr/code-server/blob/master/doc/install.md#debian-ubuntu
 
 apt-get install -y curl
 
 # Install code-server
 apt-get install -y curl
-code_server_version=${code_server_version:-3.9.2}
+last_code_server_release=$(curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/cdr/code-server/releases/latest | grep -Po 'cdr/code-server/releases/tag/v\K[^"]*')
```

Comment:
Works without the `Accept` header.
Maybe could install `jq` with `curl` and then:

```shell
curl --silent https://api.github.com/repos/cdr/code-server/releases/latest | jq --raw-output '.name | sub("^v"; "")'
```

Reply:
the VM installation process is already quite long for the python training (mainly because of pyenv + specific python version installation), I would prefer to spare the installation of a package for now. Is that OK for you?

The headers are recommended by the github documentation itself, but it's true that the command work without them. I am not sure about what to do :/

Reply:
If the doc says to use the header, better to use it.
For `jq` it's a little package that won't add much time over `curl` install. Using it is a bit safer than greping JSON as `grep` could match other lines if some other content is changed and start to match (even if it's barely possible).

Reply:
as I have only little time to upgrade the content of the Python training, I would like to focus on the content itself than on strigo ("Done is better than perfect", as @srevereault often tells me). I hope it is ok for you.